docs/Project-Board-Guide.md

```diff
@@ -0,0 +1,22 @@
+# Simorgh Project Board Guide
+
+Overview of a Github Project: https://help.github.com/en/articles/about-project-boards
+
+BBC's Simorgh Project board: https://github.com/orgs/bbc/projects/20
+
+### Overiew of each column
+
+<!-- prettier-ignore -->
+| Column Name               | Issues/Pull Requests that belong in this column | When to move card to the next column | Additional context |
+| ------------------------- | ------------------------------------------------| ------------------------------------ |------------------- |
+| To-do | Contains issues that have been newly created and added to the Simorgh project. | Once the issue has been refined to the corresponding team that has been labelled to the issue, then it can be manually moved to the "Ready for Dev". | n/a |
+| Ready for Dev | Contains issues that have been refined. | Once you have selected an issue to work on, you must assign yourself to it and manually move the issue to the "Issues in progress".| n/a |
+| Externally blocked | Contains issues/PRs that are blocked by an external factor. | Manually move the issue/PR when it is no longer blocked. | n/a |
+| Issues in progress | Contains issues that are in working progress. | If the PR that uses a [keyword to link the issue](https://help.github.com/en/articles/closing-issues-using-keywords) has been merged, then the cards in this column will move automatically to "Done". | PRs should not use keywords unless they definitely complete the entire issue. |
+| PR in progress | Contains card that have been automatically added here once a PR is assigned to the project "Simorgh". | It is your responsibility to move the card to "Code Review", when you wish for it to be reviewed by others. Teams prioritise code reviews over actual development. | PRs should be created on first commit, so that the team can see what you are working on. |
```

Comment:
```suggestion
| PR in progress | Contains cards that have been automatically added here once a PR is assigned to the project "Simorgh". | It is your responsibility to move the card to "Code Review", when you wish for it to be reviewed by others. Teams prioritise code reviews over actual development. | PRs should be created on first commit, so that the team can see what you are working on. |
```